src/current/cockroachcloud/costs.md

```diff
@@ -7,7 +7,11 @@ keywords: pricing
 
 The ways that monthly charges are incurred for a CockroachDB Cloud organization depend on the plans of its clusters: CockroachDB Basic, Standard, or Advanced. This page explains costs under each plan, specifying components that are billed based on provisioned capacity or usage.
 
-Refer to [CockroachDB Pricing](https://www.cockroachlabs.com/pricing/) for details on plans and rates.
+Refer to [CockroachDB Pricing](https://www.cockroachlabs.com/pricing/new/) for details on plans and rates.
+
+{{ site.data.alerts.callout_info }}
+This page reflects [updated costs](https://www.cockroachlabs.com/pricing/new/) recently [announced](https://www.cockroachlabs.com/blog/improved-cockroachdb-cloud-pricing/) for CockroachDB Cloud. This new pricing, including new usage-based costs, now applies to all customers except those with annual or multi-year contracts that began prior to December 1, 2024, for whom the updated pricing goes into effect upon contract renewal. Prior to renewal, line items for usage of data transfer, backups, and changefeeds are displayed in the [Billing](https://cockroachlabs.cloud/billing) interface and on invoices with a $0 charge, while showing actual usage metrics to help estimate future costs.
```

Comment:
```suggestion
This page reflects [updated costs](https://www.cockroachlabs.com/pricing/new/) recently [announced](https://www.cockroachlabs.com/blog/improved-cockroachdb-cloud-pricing/) for CockroachDB Cloud. This new pricing, including new usage-based costs, now applies to all customers except those with annual or multi-year contracts that began prior to December 1, 2024. For these customers, the updated pricing goes into effect upon contract renewal. Prior to renewal, line items for usage of data transfer, backups, and changefeeds are displayed in the [Billing](https://cockroachlabs.cloud/billing) interface and on invoices with a $0 charge, while showing actual usage metrics to help estimate future costs.
```

Reply:
![Screenshot 2024-11-27 at 2 12 59 PM](https://github.com/user-attachments/assets/c1a5d8f3-01d6-4c00-af2e-ea212b02a124)

Can we put the tiers in their own row or give Basic a little more column width so this type layout problem isn't as visible?

Reply:
Used your updated language (except "those" instead of "these", and barring any objection from the other reviewers). 

I'd tried to find a way to address the column widths while keeping the table in Markdown, recently, and couldn't find a way. Converting this one to HTML to address the issues. Thanks.